xtask/src/commands/release.rs

```diff
@@ -425,6 +427,13 @@ impl Prepare {
         {
             return Err(anyhow!("failed to generate helm docs"));
         }
+
+        replace_in_file!(
+            "./helm/chart/router/Chart.yaml",
+            "veersion: \"v\\d+.\\d+.\\d+\"",
```

Comment:
typo. veersion

Reply:
also should have been appVersion

Reply:
[b4e98f3](https://github.com/apollographql/router/pull/2202/commits/b4e98f3cb8c0b91f10d3dc0c79584e72bd881958)